client/src/pages/VerifyEmail.js

```diff
@@ -1,11 +1,107 @@
 import React from "react";
+import { connect } from "react-redux";
```

Comment:
Is there any chance you could implement `Context` instead of Redux? We are trying to move away from Redux... @robinv85 Any thoughts here?

Reply:
this example is simply getting the state that's already in redux
with the `mapStateToProps(({ session }) =>...` below
so I think it's out of scope to request refactoring the global state management for this PR :)